### DISCUSSION TITLE
Migration from Perforce

### DISCUSSION THREAD

**Discussion by srihari437** [community/community#61750] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

We are planning to migrate my project source code from perforce to GitHub enterprise.
we have 9 GB date in the perforce stream, is it possible to migrate the data with history?
FYI... am experimenting in sandbox.

please help on this

Thanks,
Srihari

**Comment by mobingood** [comment]
@srihari437 

You found any thing on it, if yes can you help me
